poetry.lock

```diff
@@ -796,7 +796,6 @@ requests = ">=2.20"
 "ruamel.yaml" = ">=0.16,<0.17.18"
 scipy = ">=1.6.0"
 sqlalchemy = {version = ">=1.4.0,<2.0.0", optional = true, markers = "extra == \"databricks\" and extra == \"redshift\""}
-sqlalchemy-redshift = {version = ">=0.8.8", optional = true, markers = "extra == \"redshift\""}
```

Comment:
Great Expectations installed without `sqlalchemy-redshift`

Reply:
This is not about file you should manually modifying

Reply:
You need to modify the poetry dependencies specified in pyproject.toml and then poetry can regenerate a new lock file.

Reply:
This likely also means removing `redshift` from the `gx extras` and specifying it separately. 
```toml
 [tool.poetry.group.sql.dependencies] 
 great-expectations = { version = "^0.18.1", extras = [ 
     "databricks", 
     "sqlalchemy1", 
     "postgres", 
 ] } 
 snowflake-sqlalchemy = ">=1.5.0" 
 snowflake-connector-python = ">=3.3.1" 
# TODO: verify the actual versions here
whatever-this-redshift-dep-is = {version: "^1.0", python: "<3.11"}
```